### DISCUSSION TITLE
dti design_matrix

### DISCUSSION THREAD

**Discussion by ZhengguoTan** [dipy/dipy#2585] [Reconstruction] [answered]
Dear dipy developers,

Thank you for sharing this excellent package. I may have a question with regard to the design_matrix function in dti.py.

https://github.com/dipy/dipy/blob/master/dipy/reconst/dti.py#L2017-L2042

The last column of the design_matrix is called dummy, which is filled with ones.

I wonder, why is this dummy column needed for DTI fitting?

Best Regards,
Zhengguo Tan

**Comment by arokem** [comment, marked as answer]
It's nicely explained (with equations) in this paper: https://pubmed.ncbi.nlm.nih.gov/16938472/

**Reply by ZhengguoTan** [reply]
Thank you. That answers my question!
